src/sqlancer/databend/DatabendErrors.java

```diff
@@ -10,12 +10,18 @@ private DatabendErrors() {
     public static void addExpressionErrors(ExpectedErrors errors) {
         errors.add("Division by zero");
         errors.add("/ by zero");
+        errors.add("ORDER BY position");
+        errors.add("GROUP BY position");
+        errors.add("downcast column error"); // bug
```

Comment:
Rather then ignoring the bugs here, could you please use the pattern described at https://github.com/sqlancer/sqlancer/blob/master/CONTRIBUTING.md#unfixed-bugs?